indico/modules/events/persons/client/js/Email.jsx

```diff
@@ -56,8 +60,13 @@ export function EmailButton({
   return (
     <>
       {!triggerSelector && (
-        <Translate as={Button} onClick={() => setOpen(true)}>
-          Send email
+        <Translate
+          as={Button}
+          onClick={() => setOpen(true)}
+          disabled={!(personIds.length || userIds.length || roleId)}
+          {...rest}
```

Comment:
I don't really like this since Translate also has some properties which aren't forwarded to the element (`as`, `context`). Maybe better to be explicit here and using `<Button><Translate></Translate></Button>`?

Reply:
It also breaks i18n extraction since `<Translate>` does not support expanded arguments (`context` could be provided through it, but this is not allowed because it needs to be known at extraction time w/o executing the code)